app/gui/src/components/TabView/SelectableTab.vue

```diff
@@ -75,10 +123,16 @@ const whenTooltip = computed(() => (props.label && !props.tooltip ? 'whenOverflo
   gap: 12px;
   transition: background-color 0.3s;
 
-  &:hover,
-  &:focus,
-  &:active {
-    background-color: var(--color-dashboard-background);
+  &.enabled {
+    &:hover,
+    &:focus,
+    &:active {
+      background-color: var(--tab-highlight);
+    }
+  }
+
+  &:not(.enabled) {
+    opacity: 0.5;
   }
```

Comment:
Prefer to not use `:not` when it can be easily avoided.
```suggestion
  opacity: 0.5;
  &.enabled {
    opacity: 1.0;
  }
```